manifests/b/BAR_Team/Beyond-All-Reason/1.1779.0/BAR_Team.Beyond-All-Reason.locale.en-US.yaml

```
@@ -0,0 +1,34 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: BAR_Team.Beyond-All-Reason
+PackageVersion: 1.1779.0
+PackageLocale: en-US
+Publisher: BAR Team
+PublisherUrl: https://www.beyondallreason.info/
+PublisherSupportUrl: https://discord.gg/N968ddE
+PrivacyUrl: https://www.beyondallreason.info/privacy
+Author: BAR Team
+PackageName: Beyond All Reason
+PackageUrl: https://github.com/beyond-all-reason/BYAR-Chobby/releases
+License: GNU General Public License v2.0
+LicenseUrl: https://raw.githubusercontent.com/beyond-all-reason/Beyond-All-Reason/master/LICENSE.md
+Copyright: Copyright © 2022 BAR Team
+CopyrightUrl: https://raw.githubusercontent.com/beyond-all-reason/Beyond-All-Reason/master/LICENSE.md
+ShortDescription: Real-Time Strategy Redefined
+Description: Free Real-Time Strategy Game with a grand scale and full physical simulation in a sci-fi setting.
+Moniker: BAR

```

```suggestion
Moniker: bar
```
Monikers are meant to be lowercase. Either `bar` or `beyondallreason` would work here.